packages/components/pages/f-mfa/src/components/Mfa.vue

```diff
@@ -2,37 +2,147 @@
     <div
         :class="$style['c-mfa']"
         data-test-id="v-mfa-component">
-        {{ copy.text }}
+        <f-card
+            :class="$style['c-mfa-card']"
+            data-test-id="f-mfa-verification-card"
+            has-inner-spacing-large
+            card-size-custom="medium"
+            has-outline>
+            <bag-surf-bg-icon
+                :class="$style['c-mfa-icon']" />
+            <h2>
+                {{ $t('verificationPage.subTitle') }}
+            </h2>
+            <p
+                v-html="$t('verificationPage.instructionsPrimaryText', { email })" />
+            <p>
+                {{ $t('verificationPage.instructionsSecondaryText') }}
+            </p>
+
+            <form
+                method="post"
+                novalidate
+                @submit.prevent="onFormSubmit">
+                <form-field
+                    v-model.lazy="verificationCode"
+                    :class="$style['c-mfa-formField']"
+                    data-test-id="mfa-verification-code-textbox"
+                    required="true"
+                    :is-visually-required=false
+                    maxlength="10"
+                    :label-text="$t('verificationPage.formField.labelText')"
+                    :placeholder="$t('verificationPage.formField.placeholderText')" />
+
+                <f-button
+                    :class="$style['c-mfa-submit']"
+                    data-test-id="mfa-submit-button"
+                    :disabled="verificationCode.trim().length < 1"
+                    button-type="primary"
+                    action-type="submit"
+                    :is-loading="isSubmitting">
+                    {{ $t('verificationPage.submitButtonText') }}
+                </f-button>
+            </form>
+
+            <f-button
+                :class="$style['c-mfa-help-info']"
```

Comment:
Should the name of this and the line below be more in sync? i.e c-mfa-need-help-link